ui/App.tsx

```diff
@@ -22,24 +24,31 @@ export default function App() {
       <ThemeProvider theme={theme}>
         <GlobalStyle />
         <Router>
-          <ErrorBoundary>
-            <AppContextProvider applicationsClient={appsClient}>
-              <Switch>
-                <Route
-                  exact
-                  path={PageRoute.Applications}
-                  component={Applications}
-                />
-                <Route
-                  exact
-                  path={PageRoute.ApplicationDetail}
-                  component={ApplicationDetail}
-                />
-                <Redirect exact from="/" to={PageRoute.Applications} />
-                <Route exact path="*" component={Error} />
-              </Switch>
-            </AppContextProvider>
-          </ErrorBoundary>
+          <AppContextProvider applicationsClient={appsClient}>
+            <Layout>
+              <ErrorBoundary>
+                <Switch>
+                  <Route
+                    exact
+                    path={PageRoute.Applications}
+                    component={Applications}
+                  />
+                  <Route
+                    exact
+                    path={PageRoute.ApplicationDetail}
+                    component={ApplicationDetail}
+                  />
+                  <Redirect exact from="/" to={PageRoute.Applications} />
+                  <Route exact path="*" component={Error} />
+                </Switch>
+              </ErrorBoundary>
+              <ToastContainer
+                position="top-center"
+                autoClose={10000}
+                newestOnTop={false}
```

Comment:
Interesting! To avoid newer things coming in and hiding messages? Might get a slightly staler error message here though, but 10s sounds unlikely.

Reply:
:thinking: I can't remember why I set that prop. I'll see what happens with multiple errors.